app/queries/occupation_standard_query.rb

```diff
@@ -15,14 +15,14 @@ def self.run(occupation_standards, search_term_params)
       .by_state_abbreviation(search_term_params[:state_abbreviation])
       .by_national_standard_type(search_term_params[:national_standard_type]&.keys)
       .by_ojt_type(search_term_params[:ojt_type]&.keys)
-      .and(
-        occupation_standards.by_title(search_term_params[:q])
-        .or(
-          occupation_standards.by_rapids_code(search_term_params[:q])
-        )
-        .or(
-          occupation_standards.by_onet_code(search_term_params[:q])
-        )
+      .where(
```

Comment:
We can mode this into a scope and only call it from here, similar to code above. This will also allow us to test this piece of code on isolation